pkg/combustion/kubernetes.go

```diff
@@ -147,9 +159,24 @@ func configureK3S(ctx *image.Context) (string, error) {
 
 		templateValues["configFile"] = k8sServerConfigFile
 		templateValues["vipManifest"] = vipManifest
+
+		return storeKubernetesInstaller(ctx, "single-node-k3s", k3sSingleNodeInstaller, templateValues)
+	}
+
+	log.Audit("WARNING: An external IP address for the Ingress controller (Traefik) must be manually configured in multi node clusters")
```

Comment:
This is only the case when `apiVip` is specified, right? Because if it's not, we should be defaulting to out of the box functionality, where klipper is still enabled, and it takes the external IP of all of the hosts. Can you confirm @atanasdinov ?

```suggestion
	log.Audit("WARNING: An external IP address for the Ingress Controller (Traefik) must be manually configured in multi-node clusters where you've specified an API VIP.")
```

Reply:
Not really. There's no multi node cluster without an apiVIP.